monitor/tests/test_views.py

```
@@ -0,0 +1,284 @@
+        
+        assert response.status_code == status.HTTP_200_OK
+        # Handle both paginated and non-paginated responses
+        if isinstance(response.data, dict) and 'results' in response.data:
+            assert len(response.data['results']) == 3
+        else:
+            assert len(response.data) == 3
+    
+    def test_latest_digest_endpoint(self):
+        """Test latest digest endpoint."""
+        # Create digests with different timestamps
+        old_digest = baker.make(
```

Ruff (0.12.2)

Local variable `old_digest` is assigned to but never used

Remove assignment to unused variable `old_digest`

(F841)